tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckAnnotations.java

```diff
@@ -103,10 +106,14 @@ public CheckCLDR handleCheck(
                 }
             }
             if (ttsMissing) {
+                final Type mainType =
+                        getPhase() != Phase.BUILD
+                                ? CheckStatus.errorType
+                                : CheckStatus.warningType; // we need to be able to check this in
```

Comment:
Sorry, it's not right

Perhaps instead:

```java
final CLDRConfig.Environment env = CLDRConfig.getInstance().getEnvironment();
final boolean onSurveyTool = env  != CLDRConfig.Environment.PRODUCTION && env != CLDRConfig.Environment.SMOKETEST;
```

then:

```suggestion
                final CLDRConfig.Environment env = CLDRConfig.getInstance().getEnvironment();
                final boolean onSurveyTool = env  != CLDRConfig.Environment.PRODUCTION && env != CLDRConfig.Environment.SMOKETEST;
                final Type mainType =
                        onSurveyTool
                                ? CheckStatus.errorType // error in ST
                                : CheckStatus.warningType; // warning elsewhere
```

Reply:
@srl295 Thanks! I have rebased locally and am trying out the code above. But `onSurveyTool ` seems to have the wrong polarity and I am getting failures running ConsolCheck. Should it not be
```
onSurveyTool = (env == CLDRConfig.Environment.PRODUCTION || env == CLDRConfig.Environment.SMOKETEST);
```
?